src/cudamatrix/cu-common.cc

```diff
@@ -67,6 +67,25 @@ void GetBlockSizesForSimpleMatrixOperation(int32 num_rows,
                "Matrix has too many rows to process");
   dimGrid->z = 1;
 }
+
+const char* cublasGetStatusString(cublasStatus_t status)
```

Comment:
thanks for the PR.
So there wasn't an official version of this function, that we could have used?

Reply:
I see that there wasn't an official function like this, but please use Google style w.r.t. open-braces: put them on the same line as the previous one, after a space, e.g.
`const char* cublasGetStatusString(cublasStatus_t status) {`

And please put a comment before the declaration of this function, saying:
`// This is analogous to the CUDA function cudaGetErrorString().`

Reply:
Reformat done.
Howerver, the CI raise some reasonless error :(